.dockerignore

```diff
@@ -1,2 +1 @@
-.git
```

Comment:
This shouldn't be removed.

Reply:
This is an artifact of the "repeat a build as a docker image" thing. Once we have `release.sh` updated to work with cross compiling (my current step), then this file will go back to ignoring everything (because we'll bind mount it in).

Reply:
We might want to keep this anyway. I don't think the release file needs to produce anything for this PR and this file is still used by the normal test build. I would feel more comfortable if we didn't remove this line and we just accepted that this was only partly done. There's plenty in this PR to begin with and it's possible for a person to manually share a volume and build it using the release dockerfile.

Reply:
I can't remove it. `pkg-config`, without git files, doesn't work. You and I figured that out together.